src/io/result_io.jl

```diff
@@ -424,12 +435,19 @@ end
 function load_results(domain::Domain)::ResultSet
     return load_results(result_location(domain))
 end
+function load_results(domain::Domain, rcps::Array{String})::ResultSet
+    return load_results(result_location(domain, rcps))
+end
```

Comment:
@ConnectedSystems ok I'll work on that.

Reply:
@ConnectedSystems Actually I was thinking/reading here and I think maybe we should change anything. Because when you are loading the results, you will call `ADRIA.load_results(<path>)` passing the path, and that way everything is going to work. This version of ` load_results(domain::Domain, rcps::Array{String})` is to be used ONLY at the end of `new_run_scenarios` function. 

An alternative would be to call 'result_location(d::Domain, rcps::Array{String})' inside 'new_run_scenarios', call `load_results(result_loc::String)` there and, by doing so, I could remove this `load_results(domain::Domain, rcps::Array{String})`.  


Also, I don't know if it makes sense to add this info (how many RCPs) to the domain because when you create the domain (when you call `ADRIA.load_domain("<path>")` you still don't know how many RCPs are going to be used.

What do you think?

Reply:
Good points, I agree.

I think then we/you just need to check that when `result_location()` is called, the ordered copy of `rcps` is used instead of the user-provided one.

One more very minor point: could you update `rcps::Array{String}` to `rcps::Vector{String}`? I can't imagine a situation where we'd expect a matrix 😛

Reply:
Hmm, also mark the `load_results(domain::Domain, rcps::Array{String})` as a private method with an underscore then, e.g.,

```
_load_results()
```

so it doesn't get exported to Main scope.